### DISCUSSION TITLE
Deep search within the lyrics

### DISCUSSION THREAD

**Discussion by serghidalg** [navidrome/navidrome#2254] [Ideas]
Hi Deluan!

First of all thank you for making such an amazing project. I came to bring an idea that I think would be easy to implement while being pretty useful when trying to look for music in big libraries.

On Spotify, they have a feature where if you input the lyrics, they search within the songs to see the one that contains that string of text.

Some of the mp3 metadata can contain lyrics information (for example if spotdl has been used). I think it would be a good idea to create a search mode where this is done so you can find the song that you are looking for without the hassle of looking it up on Google.

Thanks!
Serg

**Comment by deluan** [comment]
Thanks for the suggestion. Yes I plan to index the track's lyrics as part of #255, and was briefly discussed in https://github.com/navidrome/navidrome/issues/249#issuecomment-802048405

**Comment by NyxWinter** [comment]
Considering the recent implementation of FTS, would this now be possible by simply adding the lyrics to the fullText of a media file?

**Comment by trisk4y** [comment]
It'd be cool to keep the found lyrics (by embedded lyrics or .lrc/.txt lyrics) in a 'cleanLyrics' table header in the database when scanning.

**Comment by StephenArg** [comment]
Any idea if this feature will come soon? I was just coming to add this feature request myself. It would be cool to have
